docs/VictoriaLogs/collectors.md

```diff
@@ -0,0 +1,554 @@
+# Log collectors Setup for VictoriaLogs
+
+## Filebeat setup
+
+[Filebeat](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-overview.html) log collector supports
+[Elasticsearch output](https://www.elastic.co/guide/en/beats/filebeat/current/elasticsearch-output.html) compatible with
+VictoriaMetrics [ingestion format](https://docs.victoriametrics.com/VictoriaLogs/ingestion.html#elasticsearch-bulk-api--opensearch-bulk-api).
+
+Specify [`output.elasicsearch`](https://www.elastic.co/guide/en/beats/filebeat/current/elasticsearch-output.html) section in the `filebeat.yml`
+for sending the collected logs to VictoriaLogs:
+
+```yml
+output.elasticsearch:
+  hosts: ["http://localhost:9428/insert/elasticsearch/"]
+  parameters:
+    _msg_field: "message"
+    _time_field: "@timestamp"
+    _stream_fields: "host.hostname,log.file.path"
+```
+
+Substitute the `localhost:9428` address inside `hosts` section with the real TCP address of VictoriaLogs.
+
+The `_msg_field` parameter must contain the field name with the log message generated by Filebeat. This is usually `message` field.
+See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#message-field) for details.
+
+The `_time_field` parameter must contain the field name with the log timestamp generated by Filebeat. This is usually `@timestamp` field.
+See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#time-field) for details.
+
+It is recommended specifying comma-separated list of field names, which uniquely identify every log stream collected by Filebeat, in the `_stream_fields` parameter.
```

Comment:
a bit vise-versa

We recommend leaving it empty unless the user exactly knows the good streaming fields. We can provide examples